terragrunt/modules/crates-io/compute-static/Cargo.toml

```diff
@@ -19,6 +19,8 @@ derive_builder = "0.12.0"
 fastly = "0.9.1"
 log = "0.4.17"
 log-fastly = "0.9.1"
+once_cell = { version = "1.18.0", features = [] }
```

Comment:
Why is this syntax used? We're not enabling any optional features and afaik this does not disable default features. So why not just use `once_cell = "1.18.0"`?

Reply:
I honestly have no idea what happened here... 😂

I'll fix it :)

Reply:
now that I think about it, I think I followed https://docs.rs/regex/latest/regex/#usage and used `cargo add` for the first time. is that the default for `cargo add`!? 🤔